packages/api/src/platforms/vtex/clients/commerce/index.ts

```diff
@@ -433,6 +433,40 @@ export const VtexCommerce = (
           { storeCookies }
         )
       },
+      delete: async (reviewId: string): Promise<true> => {
+        const review = await fetchAPI(
+          `${base}/${REVIEWS_AND_RATINGS_API_PATH}/review/${reviewId}`,
+          undefined,
+          { storeCookies }
+        )
+
+        if (!review) {
+          throw new NotFoundError('Review not found')
+        }
+
+        const authCookieKey: string = `VtexIdclientAutCookie_${account}`
+
+        const authCookie = getCookieFromRequestHeaders(ctx, authCookieKey) ?? ''
+
+        if (!authCookie) {
+          throw new NotAuthorizedError('Missing auth cookie')
```

Comment:
suggestion: also not about this specific PR, but I'd suggest to change "NotAuthorizedError" to "UnauthorizedError" to match the 401 HTTP error response naming